Description: [Django 1.10]
 django.forms.util was renamed django.forms.utils
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-25

--- python-jingo-0.9.orig/jingo/monkey.py
+++ python-jingo-0.9/jingo/monkey.py
@@ -28,15 +28,15 @@ def __html__(self):
 
 
 def patch():
-    from django.forms import forms, formsets, util, widgets
+    from django.forms import forms, formsets, utils, widgets
 
     # Add __html__ methods to these classes:
     classes = [
         forms.BaseForm,
         forms.BoundField,
         formsets.BaseFormSet,
-        util.ErrorDict,
-        util.ErrorList,
+        utils.ErrorDict,
+        utils.ErrorList,
         widgets.Media,
         widgets.RadioFieldRenderer,
     ]
